usr/target.c

```diff
@@ -1332,11 +1331,15 @@ static int abort_cmd(struct target *target, struct mgmt_req *mreq,
 		 */
 
 		cmd->mreq = mreq;
-		err = -EBUSY;
-	} else {
+		eprintf("\n%s: cmd:%p\n", __func__, cmd);
+		err = cmd->dev->cmd_perform(target->tid, cmd);
```

Comment:
Do we need to still return EBUSY from here to not disturb native TGT behavior?

Reply:
abort handling return code will be returned to TGT,  TGT behaviour will not change here